Editor/Editors/CinemachineVirtualCameraBaseEditor.cs

```diff
@@ -3,6 +3,7 @@
 using System;
 using System.Collections.Generic;
 using Cinemachine.Utility;
+using UnityEngine.UIElements;
```

Comment:
Minor: why using of UIElements namespace is needed here?

Reply:
Rider accident.